docs/conf.py

```diff
@@ -64,14 +68,16 @@ def get_version(release):
 source_suffix = '.rst'
 
 # The encoding of source files.
-#source_encoding = 'utf-8-sig'
+# source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
 project = 'aiopg'
-copyright = '2014-2016, Andrew Svetlov'
+date = datetime.date.today()
+
+copyright = '2014-{year}, Andrew Svetlov'.format(year=date.year)
```

Comment:
@vir-mir I think you can add your name here (and and in all other places where it makes sense)